### PR TITLE
drop support for old django versions

### PR DESCRIPTION
**Pull request by asherf**

### CODE REVIEW

**Comment by asherf** [comment]
@dhui @rbm @adepue

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+4.0.0 (2018-02-10)
```
Comment (dhui) [inline CHANGELOG.rst]: Not sure if this change warrants a new major version.
I think it'd make sense to use a new major version if we drop Python 2 support, but not for dropping support for old Django versions.

**Comment by asherf** [comment]
@dhui good point. updated

**Comment by dhui** [comment]
:shipit:

**Pull request merged**